plugins/parsers/nagios/parser.go

```diff
@@ -25,10 +24,7 @@ func getExitCode(err error) (int, error) {
 
 	ee, ok := err.(*exec.ExitError)
 	if !ok {
-		// If it is not an *exec.ExitError, then it must be
-		// an io error, but docs do not say anything about the
-		// exit code in this case.
-		return 0, err
+		return 3, err
```

Comment:
Could we declare a constant 'unknownExit=3' or something and use it instead of the bare 3s here and later in parser.go? I understand from the guidelines doc you linked in the PR description that it's a special value for nagios but that will not be clear to people who read this code in the future.

Reply:
That is a good point. The bare value 3 is refactored into a const.